src/main/java/com/SAS/User/User.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * The class represents a user in the system
+ */
+
+package com.SAS.User;
+
+public abstract class User {
+
+    public static int counter = 0;
+
```

Comment:
you won't get the initial ID from static int, because the server will be up and down and every thime the static int will be zeroed again.

the new userID should be pulled from the DB

Reply:
You right, but we will add this change as a separate task after will add the DB.